pubsub/envelope.go

```diff
@@ -122,7 +124,7 @@ func NewCloudEventsEnvelope(id, source, eventType, subject string, topic string,
 }
 
 // FromCloudEvent returns a map representation of an existing cloudevents JSON.
-func FromCloudEvent(cloudEvent []byte, topic, pubsub, traceParent string, traceState string) (map[string]interface{}, error) {
+func FromCloudEvent(cloudEvent []byte, topic, pubsub, traceParent string, traceState string, publishTime string) (map[string]interface{}, error) {
```

Comment:
Please, create an issue to ask a refactor this so we don't need to add more to this function and can return a struct where there are methods to set additional attributes, like SetTime().